linux_os/guide/system/permissions/files/permissions_within_important_dirs/file_permissions_audit_binaries/rule.yml

```diff
@@ -13,9 +13,13 @@ description: |-
     /sbin/auditctl 755
     /sbin/aureport 755
     /sbin/ausearch 755
+    {{% if product not in ["rhel10"] %}}
     /sbin/autrace 755
+    {{% endif %}}
     /sbin/auditd 755
+    {{% if 'rhel' not in product %}}
     /sbin/audispd 755
+    {{% endif %}}
```

Comment:
This causes weird vertical spaces in HTML guide. For example, a screenshot from the the RHEL 9 CIS profile guide:

![image](https://github.com/user-attachments/assets/7496fad5-fa67-44e5-97de-68f5972058a4)


You need to put `-` after `%` or before `%` to control newlines.

Reply:
I have adjusted these, should be better now.